src/milestones.ts

```diff
@@ -47,3 +47,30 @@ export const assign = async (prNumber: number) => {
     `Set milestone ${giteaVersion.majorMinorVersion} for PR #${prNumber}`,
   );
 };
+
+export const run = async () => {
+  await removeMilestonesFromUnmergedClosedPrs();
+};
+
+// removes milestones from all PRs that are closed and unmerged
+const removeMilestonesFromUnmergedClosedPrs = async () => {
+  // get the current open milestones
+  const milestones = await github.getMilestones();
```

Comment:
We just need to get open milestones. Closed milestones could be ignored.

Reply:
Yes, `github.getMilestones()` returns the open milestones